packages/actor-init-sparql/lib/lodash.ts

```diff
@@ -0,0 +1,61 @@
+function mapValues(object: any, func: (key: any, value: any, index: number) => any): void {
```

Comment:
Great that you abstracted these functions, but this is not a good place for them.
We don't want other packages to depend on actor-init-sparql, only the other way around.

I even think most of these functions are either used once, or they can be replaced by new ES6+ functions, so this file could potentially be removed even.